src/main/java/fi/hsl/transitdata/pubtransredisconnect/Main.java

```diff
@@ -259,6 +259,14 @@ private void handleJourneyResultSet(ResultSet resultSet, Jedis jedis) throws Exc
             jedis.hmset(key, values);
             jedis.expire(key, redisTTLInSeconds);
 
+            //Insert a composite key that allows reverse lookup of the dvj id
+            //The format is route-direction-date-time
+            String joreKey = TransitdataProperties.formatJoreId(resultSet.getString(ROUTE_NAME),
+                    resultSet.getString(DIRECTION), resultSet.getString(OPERATING_DAY),
+                    resultSet.getString(START_TIME));
+            jedis.set(joreKey, resultSet.getString(DVJ_ID));
+            jedis.expire(joreKey, redisTTLInSeconds);
```

Comment:
you could use jedis.setex(joreKey, redisTTL, data). would save you one line of code :) 

hmset doesn't have this shorthand.